fm.cancel.Ripcord.yaml

```diff
@@ -1,6 +1,6 @@
 app-id: fm.cancel.Ripcord
 runtime: org.freedesktop.Platform
-runtime-version: '22.08'
+runtime-version: '23.08'
 sdk: org.freedesktop.Sdk
```

Comment:
24.08 is latest

Reply:
That pr was done last year, as the application was unmaintained I comitted my changes to my private flatpak repository instead while waiting to see if I could get some response. I will update this pr tomorrow.